components/builder-api-client/src/builder.rs

```diff
@@ -531,6 +533,136 @@ impl BuilderAPIProvider for BuilderAPIClient {
             .ok_if(&[StatusCode::NO_CONTENT])
     }
 
+    /// Accepts an origin member invitation
+    ///
+    ///  # Failures
+    ///
+    ///  * Remote builder is not available
+    ///  * Invitation does not exist
```

Comment:
After pondering your concern here for a bit, I'm reluctant to enumerate all the possible failure scenarios because, frankly, they're not quantifiable.

I have a concern that the lines under `# Failures` for each client API function really aren't providing any measurable value since there are often an untold number of ways functions could fail - ie. Builder disk loss. I think it spirit, they were meant to document the API ❤️  I believe they would provide greater value simply by expressing more verbosely the signature of the function. For example: What does the function expect for arguments? What does the function return in a happy path scenario?

Thoughts?

Reply:
I have refactored the doc comments in https://github.com/habitat-sh/habitat/pull/7349/commits/473183cd1251e9ef96b11d8c1a6cccfdac6e39d0